doajtest/unit/test_models.py

```diff
@@ -163,57 +165,26 @@ def test_04_suggestion_model_rw(self):
         s.prep()
         assert 'index' in s, s
         assert 'application_type' in s['index'], s['index']
-        assert s['index']['application_type'] == 'reapplication'
+        assert s['index']['application_type'] == 'update request'
```

Comment:
I notice that we've been a bit lax with spaces vs underscores in application **status**, though note this is application **type** and its primary usage is in the facets in the admin UI (it's a calculated field, which is why it appears in the "index" section of the object).  So, this is intended to have a space in it because it is human readable.  It isn't used anywhere else except in the UI and the tests, so nothing relies on it.

Nonetheless, you are absolutely correct, this should be in the constants module, so I've made that change.